cmd/git-sync/main.go

```diff
@@ -79,6 +79,10 @@ var flChmod = flag.Int("change-permissions", envInt("GIT_SYNC_PERMISSIONS", 0),
 var flSyncHookCommand = flag.String("sync-hook-command", envString("GIT_SYNC_HOOK_COMMAND", ""),
 	"the command executed with the syncing repository as its working directory after syncing a new hash of the remote repository. "+
 		"it is subject to the sync time out and will extend period between syncs. (doesn't support the command arguments)")
+var flSynchookCommandTimeout = flag.Duration("sync-hook-command-timeout", envDuration("GIT_SYNC_HOOK_COMMAND_TIMEOUT", time.Second),
```

Comment:
because this had no timeout before, I think the default needs to be more like 30 seconds

Reply:
set it 30, done.